webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -36,14 +36,29 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
     }, [refresh]);
 
 
-    const searchUsers = async (prefix, maxResults, resolveUserDisplayNameFN = (user => user.id)) => {
+    const searchUsers = async (prefix, after, resolveUserDisplayNameFN = (user => user.id)) => {
         let allUsersList = allUsers;
         if (allUsersList.length === 0) {
             allUsersList = await allUsersFromLakeFS(resolveUserDisplayNameFN)
             setAllUsers(allUsersList)
```

Comment:
Not your code, but this looks kinda funky - 
This sets `allUsers` on the state, which will revoke re-rendering,
While this search func continues to run.
If you have a nicer way to do this it'll be great, but not necessary for now.